kernel/kernel-defaults/src/main/java/io/delta/kernel/defaults/internal/expressions/DefaultExpressionEvaluator.java

```diff
@@ -156,6 +157,19 @@ ExpressionTransformResult visitComparator(Predicate predicate) {
             }
         }
 
+        @Override
+        ExpressionTransformResult visitNullSafeComparator(Predicate predicate) {
+            switch (predicate.getName()) {
+                case "<=>":
```

Comment:
Add the new operator to the Java doc of [Predicate](https://github.com/delta-io/delta/blob/master/kernel/kernel-api/src/main/java/io/delta/kernel/expressions/Predicate.java#L36).

Reply:
Wondering if we should just use `IS NOT DISTINCT FROM` which seems like [is in](https://github.com/delta-io/delta/blob/master/kernel/kernel-api/src/main/java/io/delta/kernel/expressions/Predicate.java#L36) the ANSI SQL.

Reply:
@vkorukanti  is`IS NOT DISTINCT FROM`  currently supported? I can't seem to find it in the source code